docs-conceptual/azps-9.2.0/configure-global-settings.md

```diff
@@ -0,0 +1,96 @@
+---
+description: Configure Azure PowerShell global settings using centralized and granular AzConfig cmdlets
+ms.custom: devx-track-azurepowershell
+ms.date: 01/09/2023
+ms.devlang: powershell
+ms.service: azure-powershell
+ms.topic: conceptual
+title: Configure Azure PowerShell global settings
+---
+
+# Configure Azure PowerShell global settings
+
+Azure PowerShell supports several global settings like disabling breaking change warning
+messages. Prior to Az PowerShell module version 9, there wasn't a centralized or granular way to
+configure global settings for Azure PowerShell.
+
+## Centralized configuration
+
+The Az PowerShell module version 9 and higher includes cmdlets for managing the configuration of
+global settings for Azure PowerShell. These `*-AzConfig` cmdlets are part of the Az.Accounts
+PowerShell module:
+
+- [Get-AzConfig][get-azconfig]
+- [Update-AzConfig][update-azconfig]
+- [Export-AzConfig][export-azconfig]
+- [Import-AzConfig][import-azconfig]
+- [Clear-AzConfig][clear-azconfig]
+
+## Granular settings
+
+You can apply settings that are only effective in a particular scope. The valid values for the
+**Scope** parameter are **MachinePolicy**, **UserPolicy**, **Process**, **CurrentUser**, and
+**LocalMachine**. **LocalMachine** is the default when applying a setting.
```

Comment:
Should we have here a link to the PowerShell documentation about scopes so someone discovering PowerShell scopes can learn more about this concept?